jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
really great idea. Would you be ok to use https://vitest.dev/ instead?

Reply:
Didn't know vitest but let's do it 👍

Reply:
Much simpler to config, it's done :)